src/Mvc/Mvc.ViewFeatures/src/DefaultHtmlGenerator.cs

```diff
@@ -51,9 +52,11 @@ public class DefaultHtmlGenerator : IHtmlGenerator
     /// <param name="urlHelperFactory">The <see cref="IUrlHelperFactory"/>.</param>
     /// <param name="htmlEncoder">The <see cref="HtmlEncoder"/>.</param>
     /// <param name="validationAttributeProvider">The <see cref="ValidationHtmlAttributeProvider"/>.</param>
+    /// <param name="mvcOptionsAccessor">The accessor for <see cref="MvcOptions"/>.</param>
     public DefaultHtmlGenerator(
         IAntiforgery antiforgery,
         IOptions<MvcViewOptions> optionsAccessor,
+        IOptions<MvcOptions> mvcOptionsAccessor,
```

Comment:
I realize this is a breaking change. Giving this type access to the `MvcOptions` in an alternate manner is something I still need to investigate.